package.json

```diff
@@ -173,6 +173,11 @@
 					"type": "boolean",
 					"description": "Controls whether Google Drive extension should alert on startup when no credentials are configured.",
 					"default": true
+				},
+				"google.drive.authPort": {
+					"type": "number",
+					"description": "Controls the port to OAuth server.",
```

Comment:
I think a better description would be something like: _Controls the port number on which a server will be started on the local machine to proceed with the OAuth2 authorization flow_

Reply:
Got it. It's really better this description. I'm going to fix it.